content/posts/2016/12/trello-zettelkasten-implementation.md

```diff
@@ -0,0 +1,293 @@
+
+Trello offers labels to help categorize your cards. Add up to 10 colored labels, and unlimited uncolored labels per board. Press `l` when viewing a card to view all available labels. You can also [filter](#filtering-cards) the board by label.
+
+<%= insert_image file: "https://trello-attachments.s3.amazonaws.com/5847acccb5e25686dc960b11/5847bcc6498b47493648b0df/7c1442f84dc19578ff75c394508fb4bc/card-labels.png", title: "Some card labels", caption: %{Some card labels} %>
+
+When adding a new card, you can type `#` plus the label’s color or title and get a list of matching labels. Use the up and down arrows to navigate the resulting list. Pressing enter or tab will add the label to the composed card. The labels will be added to the card when you submit. See [Adding labels to cards](http://help.trello.com/article/797-adding-labels-to-cards) for more details.
+
+#### Tags
+
+You can also add hash-tags to card descriptions for an extra layer of tagging. [Searching](#searching-cards) by hash-tag will show all cards containing that tag. When added to a title, you can [filter](#filtering-cards) card titles by that hash-tag.
+
+In the image below, I’ve added the tags `#essays`, `#notetaking`, `#writing` to a note title, and I am filtering the board to show only those cards with `#notetaking` in title.
+
+<%= insert_image file: "https://trello-attachments.s3.amazonaws.com/5847acccb5e25686dc960b11/5847bcc6498b47493648b0df/df20fa7c686ad435b639d94c0a3b6048/filter-hash-tag.png", title: "Filtering by hash-tag", caption: %{Filtering by hash-tag} %>
+
+### Language Support
+
+Trello is available in over 20 languages. See [Changing your language in Trello](http://help.trello.com/article/745-is-trello-available-in-other-languages).
+
+### Markdown Support
+
+Trello supports Markdown text formatting in card descriptions, checklists and comments. For more information, see [How To Format Your Text in Trello](http://help.trello.com/article/821-using-markdown-in-trello).
+
+### Mobile Apps
+
+Trello is available for Web, iPhone, iPad, Android and most mobile browsers: <https://trello.com/platforms>
+
+### More Features
+
+See [How to use Trello like a pro](http://help.trello.com/article/734-how-to-use-trello-like-a-pro) and the [Trello Resources Board](https://trello.com/b/nPNSBZjB/trello-resources) for more useful ways to work with your lists and cards.
+
+## Conclusion
+
+Trello offers a flexible set of tools to build a knowledge base. It is especially suited for collaborative note archives. However, it lacks a generic data import tool. Also, if you lose internet access you will lose access to your note archives. For these reasons, it seems Trello is best suited for small or new note archives, project-based note archives, and for people who have a constant internet connection.
+
+Let us know if you [try Trello](https://trello.com/zettel/recommend) to build a Zettel note archive. And tell us if you have any questions or suggestions to improve this article.
```

Comment:
It gives Trello Gold to the account when people use it. however I'll just remove it entirely.